Manifests/ManifestsApple/com.apple.MCX-EnergySaver.plist

```diff
@@ -155,24 +163,58 @@
 			<key>pfm_subkeys</key>
 			<array>
 				<dict>
-					<key>pfm_default</key>
-					<integer>-1</integer>
 					<key>pfm_description</key>
-					<string></string>
-					<key>pfm_hidden</key>
-					<string>all</string>
+					<string>Minutes of inactivity before the display will be turned off, in an integer value where 0 means never</string>
 					<key>pfm_name</key>
-					<string>com.apple.EnergySaver.desktop.ACPower-ProfileNumber</string>
+					<string>Display Sleep Timer</string>
+					<key>pfm_range_list</key>
+					<array>
+						<integer>0</integer>
+						<integer>1</integer>
+						<integer>2</integer>
+						<integer>3</integer>
+						<integer>5</integer>
+						<integer>10</integer>
+						<integer>20</integer>
+						<integer>30</integer>
+						<integer>45</integer>
+						<integer>60</integer>
+						<integer>75</integer>
+						<integer>90</integer>
+						<integer>105</integer>
+						<integer>120</integer>
+					</array>
+					<key>pfm_range_list_titles</key>
+					<array>
+						<string>Never</string>
+						<string>1 Minute</string>
+						<string>2 Minutes</string>
+						<string>3 Minutes</string>
+						<string>5 Minutes</string>
+						<string>10 Minutes</string>
+						<string>20 Minutes</string>
+						<string>30 Minutes</string>
+						<string>45 Minutes</string>
+						<string>1 Hour</string>
+						<string>1 Hour 15 Minutes</string>
+						<string>1 Hour 30 Minutes</string>
+						<string>1 Hour 45 Minutes</string>
+						<string>2 Hours</string>
+					</array>
```

Comment:
I completely understand the desire to be more in line with the docs, but I wonder if by adding these defined times if we're not unnecessarily limiting what's possible for admins to configure.  For someone who _might_ want to have the display go to sleep after 25 minutes, or some other custom option within the acceptable range of options, I'm hesitant to have locked in options like this.

Reply:
I may have spoken too soon ... it looks like these were in the profile to begin with.

Reply:
I'm unresolving this as it may be worth a bit more discussion.

See, I was thinking along the same lines as you - I do prefer giving admins power and granularity, and in fact in IPE all similarly ranged fields will be reverted back to free-form integer ones.

The reason that I added them here was indeed to match the already existing fields that you mentioned. But, if we're both in agreement over this, perhaps it would be better to remove all range lists from timer properties in this domain.

What do you think?

Reply:
I'd vote to remove the concrete range options.

Reply:
I removed all range lists on timer properties in commit 046487b below.